src/app/inventory/dimCsvService.factory.ts

```diff
@@ -189,9 +206,13 @@ function downloadWeapons(guns, nameMap) {
 
     data += cleanNotes(gun);
 
+    console.log(gun);
```

Comment:
And this

Reply:
That’s a hard oops times two. 

I’ll yank them in a bit.